packages/slate/src/interfaces/editor.ts

```diff
@@ -1274,7 +1275,7 @@ export const Editor = {
       at: before,
       match: Text.isText,
       reverse: true,
-      voids,
+      voids: true,
```

Comment:
I think we might want to leave this as a variable and perhaps change how `Transforms.delete` is called in slate-react instead.